config/routes.rb

```diff
@@ -11,6 +11,10 @@
   get "home/index"
   get "reports/index"
 
+  patch "stories/:id/approve", to: "stories#approve", as: "approve_story"
+  patch "stories/:id/reject", to: "stories#reject", as: "reject_story"
+  patch "stories/:id/pending", to: "stories#pending", as: "pending_story"
```

Comment:
this could be refactored to let rails add most of the boilerplate https://guides.rubyonrails.org/routing.html#adding-member-routes

moving this to the next block like:

```rb
resource :stories do
  member do
    patch :approve
    patch :reject
    patch :pending
  end
  ...
  ...
```

I think it creates the same routes